assets/scripts/src/choices.js

```diff
@@ -2348,6 +2347,14 @@ class Choices {
       dropdown.insertBefore(input, dropdown.firstChild);
     }
 
+    if (
+      this.placeholder &&
+      this.passedElement.type !== 'select-one' &&
+      !this.config.searchPlaceholderValue
+    ) {
+      input.style.width = getWidthOfInput(input);
```

Comment:
Imagine I got three selects with different sizing `.select--large`, `.select--small` and without class for base size. Without `setTimeout` delay it always returns styling of base select size. With `setTimeout` it returns correct values. It looks like that css applies slower in larger codebase and we need to wait little bit for that.